proto/wippersnapper/i2c/v1/i2c.proto

```diff
@@ -151,33 +151,35 @@ message I2CDeviceDeinitResponse {
 * SensorType allows us determine what types of units the sensor uses, etc.
 */
 enum SensorType {
-  SENSOR_TYPE_UNSPECIFIED         = 0;
-  SENSOR_TYPE_ACCELEROMETER       = 1; /** Gravity + linear acceleration */
-  SENSOR_TYPE_MAGNETIC_FIELD      = 2;
-  SENSOR_TYPE_ORIENTATION         = 3;
-  SENSOR_TYPE_GYROSCOPE           = 4;
-  SENSOR_TYPE_LIGHT               = 5;
-  SENSOR_TYPE_PRESSURE            = 6;
-  SENSOR_TYPE_PROXIMITY           = 8;
-  SENSOR_TYPE_GRAVITY             = 9;
-  SENSOR_TYPE_LINEAR_ACCELERATION = 10; /** Acceleration not including gravity */
-  SENSOR_TYPE_ROTATION_VECTOR     = 11;
-  SENSOR_TYPE_RELATIVE_HUMIDITY   = 12;
-  SENSOR_TYPE_AMBIENT_TEMPERATURE = 13;
-  SENSOR_TYPE_OBJECT_TEMPERATURE  = 14;
-  SENSOR_TYPE_VOLTAGE             = 15;
-  SENSOR_TYPE_CURRENT             = 16;
-  SENSOR_TYPE_COLOR               = 17;
-  SENSOR_TYPE_RAW                 = 18;
-  SENSOR_TYPE_PM10_STD            = 19; /** Standard Particulate Matter 1.0 */
-  SENSOR_TYPE_PM25_STD            = 20; /** Standard Particulate Matter 2.5 */
-  SENSOR_TYPE_PM100_STD           = 21; /** Standard Particulate Matter 100 */
-  SENSOR_TYPE_PM10_ENV            = 22; /** Environmental Particulate Matter 1.0 */
-  SENSOR_TYPE_PM25_ENV            = 23; /** Environmental Particulate Matter 2.5 */
-  SENSOR_TYPE_PM100_ENV           = 24; /** Environmental Particulate Matter 100 */
-  SENSOR_TYPE_CO2                 = 25; /** CO2 */
-  SENSOR_TYPE_GAS_RESISTANCE      = 26; /** The resistance (in Ohms) of the gas sensor.  This is proportional to the amount of VOC particles in the air. */
-  SENSOR_TYPE_ALTITUDE            = 27; /** The altitude in meters. */
+  SENSOR_TYPE_UNSPECIFIED         = 0;  /** Sensor value type which is not defined by this list, error. */
+  SENSOR_TYPE_ACCELEROMETER       = 1;  /** Acceleration, in meter per second per second. */
```

Comment:
redundant "meters per second per second" i think

Reply:
Not redundant - https://en.wikipedia.org/wiki/Metre_per_second_squared